corehq/apps/translations/generators.py

```diff
@@ -24,6 +28,88 @@
 SKIP_TRANSFEX_STRING = "SKIP TRANSIFEX"
 
 
+class EligibleForTransifexChecker(object):
```

Comment:
This is a great addition.
how about moving it to a file of its own and under the 'integrations.transifex' since it relates that.
so then this would become
`corehq.apps.translations.integrations.transifex.checker.EligibleForTransifex`, also dropping the Checker in the end.